python/ray/util/state/state_cli.py

```diff
@@ -183,6 +183,7 @@ def output_with_format(
             explicit_start=True,
             # We want to keep the defined ordering of the states, thus sort_keys=False
             sort_keys=False,
+            explicit_end=True,
```

Comment:
what's this for?

Reply:
Primarily, to split the result output and confirmed it's a no-op